tests/phpunit/tests/strings-in-block/test-attributes.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+namespace WPML\PB\Gutenberg\StringsInBlock;
+
+/**
+ * @group page-builders
+ * @group gutenberg
+ * @group strings-in-block
+ */
+class TestAttributes extends \OTGS_TestCase {
+
+	const BLOCK_NAME = 'block/type';
+
+	/**
+	 * @test
+	 */
+	public function it_should_find() {
```

Comment:
@strategio It should find what? Please be specific when naming tests.

Reply:
@andreasciamanna, I created another PR https://github.com/OnTheGoSystems/wpml-page-builders-gutenberg/pull/41 to introduce the parameter `search-method`.

Reply:
Also fixed the test method names.

Reply:
@strategio it's still called `it_should_find`, isn't it?

Reply:
It's fixed in the other PR Andrea.